lisa/tests/base.py

```diff
@@ -873,7 +889,8 @@ def dmesg_entries(self):
                 if line.strip()
             ]
 
-    def test_dmesg(self, level='warn', facility=None) -> ResultBundle:
+    def test_dmesg(self, level='warn', facility=None, pattern_whitelist:DmesgTestConf.PatternWhitelist=None) -> ResultBundle:
```

Comment:
Nit: `pattern_ignore` would be more explicit. I was a bit confused at first (i.e. does having a whitelist mean we'll only consider messages matching the pattern, or will we let messages matching the pattern "pass"?).

Reply:
I'll go for `ignore_patterns` so it shows that it's a list (or at least an iterable).